examples/karpenter/main.tf

```diff
@@ -104,6 +107,9 @@ module "karpenter_irsa" {
   attach_karpenter_controller_policy = true
 
   karpenter_controller_cluster_id = module.eks.cluster_id
+  karpenter_controller_ssm_parameter_arns = [
```

Comment:
we should also update the version on line 104 to reflect the recent partition changes (i.e. - set to `"~> 4.21.1"`)